config/global.ini.php

```diff
@@ -663,6 +663,12 @@
 ; - links to Uninstall themes will be disabled (but user can still enable/disable themes)
 enable_plugins_admin = 1
 
+; By setting this option to 0 the users management will be disabled
+enable_users_admin = 1
+
+; By setting this option to 1 the websites management will be disabled
```

Comment:
Typo here, I think. `1` should be `0`, correct?

Reply:
Yes 👍 fixed